<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SweepWidget</class>
 <widget class="QWidget" name="SweepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_7">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Sweep Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QGroupBox" name="smuA_grp">
             <property name="title">
              <string>SMU A</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="2">
               <widget class="QComboBox" name="aTypeBox">
                <item>
                 <property name="text">
                  <string>Voltage</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Current</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Constant</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QDoubleSpinBox" name="aStartVal">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>200.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="aStartLabel">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QComboBox" name="aIntTimeBox">
                <item>
                 <property name="text">
                  <string>Auto</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Low</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mid</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>High</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QDoubleSpinBox" name="aStepVal">
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>200.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Step</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>59</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QDoubleSpinBox" name="aEndVal">
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>200.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Integration Time</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QDoubleSpinBox" name="aLimitVal">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>200.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Limit</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="smuB_grp">
             <property name="title">
              <string>SMU B</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="3" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QComboBox" name="bIntTimeBox">
                <item>
                 <property name="text">
                  <string>Auto</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Low</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mid</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>High</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QDoubleSpinBox" name="bStepVal">
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>200.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>104</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="2">
               <widget class="QComboBox" name="bTypeBox">
                <item>
                 <property name="text">
                  <string>Voltage</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Current</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Constant</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Integration Time</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QDoubleSpinBox" name="bEndVal">
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>200.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Step</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="bStartLabel">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QDoubleSpinBox" name="bStartVal">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>200.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QDoubleSpinBox" name="bLimitVal">
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="decimals">
                 <number>3</number>
                </property>
                <property name="maximum">
                 <double>200.000000000000000</double>
                </property>
                <property name="value">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Limit</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Data Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Y- Axis</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>SMU Channel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="2">
               <widget class="QComboBox" name="axisYSMU">
                <item>
                 <property name="text">
                  <string>SMU A</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../materials.qrc">
                   <normaloff>:/materials/materials/letter-a.png</normaloff>:/materials/materials/letter-a.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>SMU B</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../materials.qrc">
                   <normaloff>:/materials/materials/letter-b.png</normaloff>:/materials/materials/letter-b.png</iconset>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Meas Type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QComboBox" name="axisYType">
                <item>
                 <property name="text">
                  <string>Voltage</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Current</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Resistance</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Power</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>X- Axis</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>SMU Channel</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="2">
               <widget class="QComboBox" name="axisXSMU">
                <item>
                 <property name="text">
                  <string>SMU A</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../materials.qrc">
                   <normaloff>:/materials/materials/letter-a.png</normaloff>:/materials/materials/letter-a.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>SMU B</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../materials.qrc">
                   <normaloff>:/materials/materials/letter-b.png</normaloff>:/materials/materials/letter-b.png</iconset>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Meas. Type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QComboBox" name="axisXType">
                <item>
                 <property name="text">
                  <string>Voltage</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Current</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Resistance</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Power</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>116</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="StopBtn">
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="../materials.qrc">
          <normaloff>:/materials/materials/cross.png</normaloff>:/materials/materials/cross.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="runBtn">
        <property name="text">
         <string>Run</string>
        </property>
        <property name="icon">
         <iconset resource="../materials.qrc">
          <normaloff>:/materials/materials/check.png</normaloff>:/materials/materials/check.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="canvasLayout"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../materials.qrc"/>
 </resources>
 <connections/>
</ui>
